src/pages/Antiquities/Antiquities.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { Accordion, Avatar, Container, Group, Text, Title } from '@mantine/core';
+import locationsJson from '@sources/locations.json';
+import antiquitiesJson from '@sources/antiquities.json';
+import classes from './Antiquities.css';
+import { AntiquityCard } from '@/components/AntiquityCard/AntiquityCard';
+import { Antiquity, Location } from '@/@typings/database-types';
+
+export function Antiquities() {
+  const locations: Location[] = locationsJson;
+  const antiquities: Antiquity[] = antiquitiesJson;
+
+  return (
+    <Container size="sm" className={classes.container}>
+      <Title order={1} className={classes.title}>
+        Antiquities
+      </Title>
+      <Accordion>
+        {locations.map((location) => (
+          <Accordion.Item key={location.name} value={location.name}>
+            <Accordion.Control>
+              <AccordionLabel label={location.name} avatar={location.avatar} />
+            </Accordion.Control>
+            <Accordion.Panel>
+              <Accordion>
+                {location.areas.map((area) => (
+                  <Accordion.Item key={area.name} value={area.name}>
+                    <Accordion.Control>
+                      <AccordionLabel label={area.name} avatar={area.avatar} />
+                    </Accordion.Control>
+                    <Accordion.Panel>
+                      <Accordion>
+                        {location.areas.map((antiquityArea) => {
```

Comment:
Those maps and filters look like a performance issue. What is the complexity of the current implementation in terms of antiquities? Can it be done better?